data/contributors.js

```diff
@@ -1,4 +1,14 @@
-const contributors = [{
```

Comment:
Please remove this file and make a new Updated PR and please add your data at the end of the list.